medical/models/medical_physician.py

```diff
@@ -111,102 +107,3 @@ def create(self, cr, uid, vals, context=None):
         vals['groups_id'] = [(6, 0, group_ids)]
```

Comment:
Hello @lasley ,
It is no longer used since the patient became a res.partner instead of a res.users